### DISCUSSION TITLE
sec/vulns/log4j/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#41] [General]
# sec/vulns/log4j/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/sec/vulns/log4j/

**Comment by KaleidScoper** [comment]
不愧是ZJU的佬

**Comment by Moyu117** [comment]
羡慕 不愧是浙大 课程的设计真的做的很好 大一就能有这么深入的探讨
